internal/pkg/vm/internal/vmcontext/testing.go

```diff
@@ -42,6 +52,21 @@ var _ vstate.VMWrapper = (*ValidationVMWrapper)(nil)
 var _ vstate.Applier = (*ValidationApplier)(nil)
 var _ vstate.KeyManager = (*KeyManager)(nil)
 
+var ChainvalActors = dispatch.NewBuilder().
+	Add(builtin.InitActorCodeID, &init_.Actor{}).
+	Add(builtin.AccountActorCodeID, &account.Actor{}).
+	Add(builtin.MultisigActorCodeID, &multisig.Actor{}).
+	Add(builtin.PaymentChannelActorCodeID, &paych.Actor{}).
+	Add(builtin.StoragePowerActorCodeID, &power.Actor{}).
+	Add(builtin.StorageMarketActorCodeID, &market.Actor{}).
+	Add(builtin.StorageMinerActorCodeID, &miner.Actor{}).
+	Add(builtin.SystemActorCodeID, &system.Actor{}).
+	Add(builtin.RewardActorCodeID, &reward.Actor{}).
+	Add(builtin.CronActorCodeID, &cron.Actor{}).
```

Comment:
I notice the verified data registry actor is missing here. Would you mind proposing the specs-actors PR to add the list of default actors and their addresses?

Reply:
Did you mean their _codeIDs_?

Reply:
PR here: https://github.com/filecoin-project/specs-actors/pull/386